src/axom/core/execution/internal/omp_exec.hpp

```diff
@@ -48,7 +48,7 @@ struct execution_space< OMP_EXEC >
   static int allocatorID() noexcept
   {
 #ifdef AXOM_USE_UMPIRE
-    return axom::getResourceAllocatorID(umpire::resource::Host);
+    return axom::getUmpireResourceAllocatorID(umpire::resource::Host);
 #else
     return 0;
```

Comment:
These should return ``axom::DEFAULT_ALLOCATOR_ID`` instead of ``0``

Reply:
Good suggestion. Done.